daemon/cmd/datapath.go

```diff
@@ -403,6 +404,12 @@ func (d *Daemon) initMaps() error {
 		}
 	}
 
+	if !option.Config.EnableIPv4FragmentsTracking {
+		if err := fragmap.InitMap(); err != nil {
```

Comment:
Is this meant to be reversed?

```suggestion
	if option.Config.EnableIPv4FragmentsTracking {
		if err := fragmap.InitMap(); err != nil {
```

Reply:
In practice this map will be created during the datapath load anyway so it's not critical which probably explains why the tests can pass, but good to fix up.

Reply:
Ouch, nice catch! Thanks.